app/package-lock.json

```diff
@@ -2226,6 +2226,15 @@
       "resolved": "https://registry.npmjs.org/@tmcw/togeojson/-/togeojson-4.2.0.tgz",
       "integrity": "sha512-kjUJemmidx4SS9QT4P5nUkS6s9IE0xpqwaWcnDd7q0McFhfRiAHJMEVarxcFXecvZAIyPt9+974NfOGr7/cKfg=="
     },
+    "@turf/bbox": {
```

Comment:
question:
I thought we were not committing changes to lockfile, what's the standard on this? :)

Reply:
hmmm... I thought the package-lock was in the .gitignore